src/parser/github-comment-module.ts

```diff
@@ -298,7 +298,7 @@ export class GithubCommentModule implements Module {
           </h6>
         </b>
       </summary>
-      ${result.feeRate !== undefined ? `<h6>⚠️ ${new Decimal(result.feeRate).mul(100)}% fee rate has been applied. Consider using the&nbsp;<a href="https://dao.ubq.fi/dollar" target="_blank" rel="noopener">Ubiquity Dollar</a>&nbsp;for no fees.</h6>` : ""}
+      ${result.feeRate !== undefined ? `<h6>⚠️ ${result.feeRate}% fee rate has been applied. Consider using the&nbsp;<a href="https://dao.ubq.fi/dollar" target="_blank" rel="noopener">Ubiquity Dollar</a>&nbsp;for no fees.</h6>` : ""}
```

Comment:
Missing whitespaces around the `</a>`

Reply:
I used &nbsp; which creates the space because adding a normal whitespace would be useless since we are using html tags and raw text. it will just join together

Reply:
Oh my bad I didn't see them, all good.